torch_geometric/utils/scatter.py

```diff
@@ -117,6 +116,13 @@ def scatter(src: Tensor, index: Tensor, dim: int = 0,
 
         raise ValueError(f"Encountered invalid `reduce` argument '{reduce}'")
 
+    @torch._dynamo.optimize()
+    def _scatter_min_or_max(src: Tensor, index: Tensor, dim: int, size: int,
+                            reduce: str):
+        return src.new_zeros(size).scatter_reduce_(dim, index, src,
+                                                   reduce=f'a{reduce}',
+                                                   include_self=False)
```

Comment:
@rusty1s Disabling this part of the computation will address the accuracy issue.